pennylane/templates/layers/strongly_entangling.py

```diff
@@ -235,3 +236,47 @@ def shape(n_layers, n_wires):
         """
 
         return n_layers, n_wires, 3
+
+    # pylint:disable = no-value-for-parameter
+    @staticmethod
+    def compute_plxpr_decomposition(*args, **hyperparameters):
```

Comment:
Can we be explicit here about what `args` and `hyperparameters` are?

```suggestion
    def compute_plxpr_decomposition(weights, wires, ranges=None, imprimitive=None):
```

Reply:
Indeed, updated!